changes-set.txt

```diff
@@ -83,6 +83,8 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+28-Dec-23 domep     dmep        moved from SF's mathbox to main set.mm
+28-Dec-23 compel    velcomp     moved from AS's mathbox to main set.mm
```

Comment:
missing entry: eqri moved from TA's mathbox to main set.mm

Reply:
The label didn't change.

Reply:
changes-set.txt should also contain theorems which are moved from mathboxes to main without label changes, see (lots of) entries with [same] in column "New".